Assets/Scripts/EnemyController.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyController : MonoBehaviour
+{
+    public float enemySpeed;
+    public int movingRight = 1;
+    public GameObject groundDetector;
+    public float rayDistance;
+    public Animator enemyAnimator;
+    // public PlayerController playerController;
+    // public PlayerHealthController playerHealthController;
+    // public PlayerHealthDisplayController playerHealthDisplayController;
+
+    private void Update()
+    {
+        PatrolEnemy();
+    }
+
+    private void PatrolEnemy()
+    {
+        enemyAnimator.SetBool("IsPatrol", true);
```

Comment:
this particular line needs to be executed only once rather than every frame. You want to set your enemy animation to move , so you can do this in start or awake. No need to set to true again again when it is already true.